src/libs/ajax/leonardo/Apps.ts

```diff
@@ -111,6 +111,13 @@ export const Apps = (signal: AbortSignal) => ({
       _.mergeAll([authOpts(), appIdentifier, { signal, method: 'DELETE' }])
     );
   },
+  getAppV2: (appName: string, workspaceId: string): Promise<GetAppResponse> => {
+    const res = fetchLeo(
+      `api/apps/v2/${workspaceId}/${appName}`,
+      _.mergeAll([authOpts(), appIdentifier, { signal, method: 'GET' }])
+    );
+    return res.json();
```

Comment:
Okay, super naive question on my part, but why don't we need to await in this case?

Reply:
Also do you need to add `workspaceId: string | null;` to the `GetAppResonse` object in app-models.ts?

Reply:
We want the caller to await the json return. I'm not adding the workspaceId field in yet since its not in prod yet and the UI releases on a more regular cadence. I will make a subtask to do this and remove the use of `ListAppResponse` inn method signatures over App that this PR introduces, which this will resolve.

Reply:
Gotcha, thanks! If you can link the subtask in the code with a TODO that would be great :)